android/envoy/src/main/kotlin/org/greatfire/envoy/EnvoyInterceptor.kt

```diff
@@ -127,7 +128,7 @@ class EnvoyInterceptor : Interceptor {
             with (builder) {
                 addHeader("Host-Orig", url.host)
                 addHeader("Url-Orig", url.toString())
-                url(state.activeService!!.url + "?test=" + t)
+                url(state.activeService!!.envoyUrl + "?test=" + t)
```

Comment:
What's going on here? Why does EnvoyTest have an envoyUrl now?

Reply:
i believe this was a byproduct of something i ran into where several private get___() methods in EnvoyTest had to be made public/protected and then they triggered the error where there was a get method with no corresponding variable.  it looks like i renamed getEnvoyUrl though, so i'll see if i can revert this.

Reply:
`envoyUrl` is such a vague term... here it means "URL to an Envoy proxy" though the `envoyUrl` you removed in the demo app means "URL that uses the envoy:// scheme". I realized I had eliminated every other variable in the project called "envoy_url" or "envoyUrl" 😆

Reply:
i reverted this to its original name, whatever was causing the conflict seems to have been changed or removed.